spec/views/accounts/new.html.haml_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'accounts/new.html.erb', type: :view do
+  context 'anonymous user' do
+    before do
+      root_request
+      page.visit new_account_url
+    end
+
+    it 'renders a new account form' do
+      expect(find_field('Name').visible?).to eq true
```

Comment:
Reallly weak sauce view specs. But in the primordial place we are now, its more a placeholder/proof of setup.